doc/metadata_routing.rst

```diff
@@ -61,88 +83,76 @@ in this section require the following imports and data::
 Weighted scoring and fitting
 ----------------------------
 
-Here :class:`~model_selection.GroupKFold` requests ``groups`` by default. However, we
-need to explicitly request weights for our scorer and the internal cross validation of
-:class:`~linear_model.LogisticRegressionCV`. Both of these *consumers* know how to use
-metadata called ``sample_weight``::
+:class:`~model_selection.GroupKFold` requests ``groups`` by default. However, we need to
+explicitly request weights for our scorer and the internal cross validation of
+:class:`~linear_model.LogisticRegressionCV` by specifying `sample_weight=True`. Both of
+these :term:`consumers <consumer>` know how to use metadata called ``sample_weight``::
 
-  >>> weighted_acc = make_scorer(accuracy_score).set_score_request(
-  ...     sample_weight=True
-  ... )
-  >>> lr = LogisticRegressionCV(
-  ...     cv=GroupKFold(), scoring=weighted_acc,
-  ... ).set_fit_request(sample_weight=True)
+  >>> weighted_acc = make_scorer(accuracy_score).set_score_request(sample_weight=True)
+  >>> lr = LogisticRegressionCV(cv=GroupKFold(), scoring=weighted_acc,)
+  >>> lr.set_fit_request(sample_weight=True)
   >>> cv_results = cross_validate(
   ...     lr,
   ...     X,
   ...     y,
-  ...     props={"sample_weight": my_weights, "groups": my_groups},
+  ...     params={"sample_weight": my_weights, "groups": my_groups},
   ...     cv=GroupKFold(),
   ...     scoring=weighted_acc,
   ... )
 
 Note that in this example, ``my_weights`` is passed to both the scorer and
 :class:`~linear_model.LogisticRegressionCV`.
 
-Error handling: if ``props={"sample_weigh": my_weights, ...}`` were passed
+Error handling: if ``params={"sample_weigh": my_weights, ...}`` were passed
 (note the typo), :func:`~model_selection.cross_validate` would raise an error,
 since ``sample_weigh`` was not requested by any of its underlying objects.
 
 Weighted scoring and unweighted fitting
 ---------------------------------------
 
 When passing metadata such as ``sample_weight`` around, all ``sample_weight``
-:term:`consumers <consumer>` require weights to be either explicitly requested
-or not requested (i.e. ``True`` or ``False``) when used in another
-:term:`router` such as a :class:`~pipeline.Pipeline` or a ``*GridSearchCV``. To
-perform an unweighted fit, we need to configure
-:class:`~linear_model.LogisticRegressionCV` to not request sample weights, so
+:term:`consumers <consumer>` require weights to be either explicitly requested or not
+requested (i.e. ``True`` or ``False``). Thus, to perform an unweighted fit, we need to
```

Comment:
the "when used in another
:term:`router` such as a :class:`~pipeline.Pipeline` or a ``*GridSearchCV``" is rather important, why remove it? Consumers don't need to request anything if they're not used in another router.

Reply:
the significance of that part here is that the user doesn't need to set any requests if they are not going to put those estimators and scorers in a router. Users can always do `LogisticRegression().fit(X, y, sample_weight)` w/o setting any requests. How do you want to convey that message here?

Reply:
I was thinking of the "passing metadata round" phrase as passing it into a router.

What about:
```
When passing metadata such as ``sample_weight`` into a :term:`router`
(:term:`meta-estimators` or routing function), all ``sample_weight`` :term:`consumers
<consumer>` require weights to be either explicitly requested or explicitly not
requested (i.e. ``True`` or ``False``).
```